dali/ft/filecopy.cpp

```diff
@@ -497,17 +502,20 @@ int FileSizeThread::run()
         RemoteFilename remoteFilename;
         for (;;)
         {
-            cur.clear();
-            cs.enter();
-            if (queue.ordinality())
-                cur.setown(&queue.popGet());
-            cs.leave();
+            {
+                cur.clear();
```

Comment:
The cur.clear() is still a problem, as it's outside the crit.

Reply:
Moved into.